prow/plugins/label/label.go

```diff
@@ -78,7 +78,36 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 	return
 }
 
-func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) error {
+func getLabelsFromGenericMatches(matches [][]string, allowedLabels []plugins.Label) (labels []string) {
+	if len(allowedLabels) == 0 {
+		return nil
+	}
+out:
+	for _, match := range matches {
+		parts := strings.Split(match[0], " ")
```

Comment:
Can't we add a capturing group for the command and not need this, feels like the input here should already be processed

Reply:
We already process labels like this so I prefer not to fix it here.